lib/posix/pthread_mutex.c

```diff
@@ -112,18 +116,20 @@ static int acquire_mutex(pthread_mutex_t *mu, k_timeout_t timeout)
 		return EINVAL;
 	}
 
+	bit = posix_mutex_to_offset(m);
```

Comment:
seems like this name is inaccurate; it's not the bit number, it's the index into the static array (yes, they're the same value, but used in different contexts, I think?)

Reply:
It's the bit in the bit array.